src/mpyl/project.py

```diff
@@ -304,6 +307,17 @@ def from_config(values: dict):
         )
 
 
+@dataclass(frozen=True)
+class Dagster:
+    repo: str
+
+    @staticmethod
+    def from_config(values: dict):
+        return Dagster(
+            repo=values.get("repo", {}),
+        )
```

Comment:
Is repo mandatory? If so then I'd say do
```suggestion
    @staticmethod
    def from_config(values: dict):
        repo = values.get("repo")
        if not repo:
          raise KeyError("Dagster config needs to have repo field set")
        return Dagster(
            repo=repo
        )
```

Reply:
done in
[34b4331](https://github.com/Vandebron/mpyl/pull/146/commits/34b4331b34b4fa1ada15dd88ce7115eb7e08464c)